src/vscode-dts/vscode.d.ts

```diff
@@ -11737,7 +11737,7 @@ declare module 'vscode' {
 		 * until `Terminal.show` is called. The typical usage for this is when you need to run
 		 * something that may need interactivity but only want to tell the user about it when
 		 * interaction is needed. Note that the terminals will still be exposed to all extensions
-		 * as normal. And the hidden terminals won't be restored when the workspace is next opened.
+		 * as normal and they will remain hidden when the workspace is reloaded.
```

Comment:
@Tyriar Sorry for the late comment.

Honestly, I don't think your changes have the same meaning as mine. These hidden terminals will not be restored at all, rather being restored and remain hidden. When I had 2 terminals opened, 1 hidden and 1 not, there would be only 1 terminal opened when the workspace is reloaded.

I used `vscode.window.terminals.length` to verify it, which is exactly why I want to add this comment, telling developers these hidden terminals that cannot be closed manually by users won't be restored, no need to worry about that there would be tons of hidden terminal when  the workspace is reloaded.

I'm not sure if I expressed it correctly. Looking forward for your reply. Thanks.

Reply:
Thanks a bunch for following up and sorry about the delayed response 😅 

I brought back your correct description in https://github.com/microsoft/vscode/pull/207016